Documentation/op-guide/performance.md

```diff
@@ -10,4 +10,26 @@ There are other sub-systems which impact the overall performance of etcd. Each s
 
 ## Benchmarks
 
-TODO
\ No newline at end of file
+To benchmark an indivisual package, etcd has Go benchmark tests. For example, mvcc package has `BenchmarkWatchableStoreWatchSyncPut` to measure how fast multiple synced watchers can receive events. To learn more about `go test` command, please visit [testing](https://golang.org/pkg/testing/). To benchmark etcd performance, simply use [benchmark](https://github.com/coreos/etcd/tree/master/tools/benchmark) CLI tool. Following results are run with:
+
+- Google Cloud Compute Engine
+- 3 machines of 8 vCPUs + 16GB Memory + 50GB SSD
+- 1 machine(client) of 16 vCPUs + 30GB Memory + 50GB SSD
+- Ubuntu 15.10
+- etcd v3 master branch, Go master branch as of May 24th, 2016
+
+Write:
+
+| Key size in bytes | Value size in bytes | Number of connections | Number of clients | Target etcd server | Average write QPS |
+|-------------------|---------------------|-----------------------|-------------------|--------------------|-------------------|
+| 8 | 256 | 100 | 1000 | leader only | 40,000 |
+| 8 | 256 | 100 | 1000 | all servers | 30,000 |
+
+Read:
+
+| Key size in bytes | Value size in bytes | Number of connections | Number of clients | Consistency | Average read QPS |
+|-------------------|---------------------|-----------------------|-------------------|-------------|------------------|
+| 8 | 256 | 100 | 1000 | Linearized | 45,000 |
+| 8 | 256 | 100 | 1000 | Serialized | 55,000 |
```

Comment:
s-read per member is even slow that writes that only sent to leader (which is 30,000). so there must be an issue either how you benchmark it or in etcd itself.